CHANGELOG.md

```diff
@@ -20,8 +20,9 @@ For information on changes in released versions of Teku, see the [releases page]
 - `--ws-checkpoint` CLI now accepts a URL optionally, and will load the `ws_checkpoint` field from that URL.
 - validator-client now publishes `validator_current_epoch` which is the epoch based on slot events on the validator client.
 - Reduced CPU usage by avoiding creation of REST API events when there are no subscribers.
-- Added a labelled counter to metrics for external signer requests, `validator_external_signer_requests`, with a result label containing `success`, `failed`, `timeout`
+- Added a labelled counter to metrics for external signer requests, `validator_external_signer_requests`, with labels `success`, `failed`, `timeout`
 - Added a labelled counter to metrics for storing the results of duties, `validator_duties_performed`, with a `type` and `result`.
+- Added a labelled counter to metrics for event stream, `validator_event_stream`, which has a `reason` label. The reason will be one of (disconnect, connect, error, timeout, invalidEvent).
```

Comment:
It might be worth combining these new metrics into a single changelog "Added new validator metrics" item with a sub-list showing each new metric.